og_ui/og_ui.module

```diff
@@ -295,11 +295,16 @@ function og_ui_menu() {
     'page arguments' => array('og_ui_field_settings'),
     'title' => 'OG field settings',
     'access arguments' => array('administer group'),
-    'description' => 'Configure Organic groups fields in bundles.',
+    'description' => 'Manage Organic groups fields.',
     'file' => 'og_ui.admin.inc',
     'type' => MENU_NORMAL_ITEM,
   );
 
+  // $items['admin/config/group/fields/add'] = array(
+  //   'title' => 'OG field settings',
+  //   'type' => MENU_DEFAULT_LOCAL_TASK,
+  // );
+
```

Comment:
These commented lines above are the remnant of an experiment I did and then decided to remove. These commented lines should be removed in the PR

Reply:
Fixed.